allennlp/models/encoder_decoders/wikitables_semantic_parser.py

```diff
@@ -1117,6 +1126,14 @@ def take_step(self,
             # entity_action_mask: (group_size, num_entity_actions)
             entity_action_logits, entity_action_mask, entity_type_embeddings = \
                     self._get_entity_action_logits(state, actions_to_link, attention_weights)
+
+            # The entity and action logits are combined with a mixture weight to prevent the
```

Comment:
I'd put this behind a flag - allow the `mixture_feedforward_params` to be `None`, and if they are, you skip the mixture.  That will let us try both versions side-by-side with just a configuration change, instead of a code change.

Reply:
done.